src/Uno.UI.Toolkit/ElevatedView.Android.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+
+namespace Uno.UI.Toolkit
+{
+	partial class ElevatedView : Border
```

Comment:
in one file it inherits from Border and the other from FrameworkElement?

Reply:
Border is sealed in WinUI, so we should avoid that.

Reply:
This is intended. The elevated view on Android is behaving exactly as a border with an elevation set to it.  But it's not the case on iOS.  I fail to see how it can be a problem to have different base on different platform, specially when the recipe for this (the elevation shadow) is very different.

Reply:
It will be a problem when we make `Border` sealed to match WinUI.